### DISCUSSION TITLE
kumiko for frame detection

### DISCUSSION THREAD

**Discussion by mizaki** [GeoRW/ACBF-Editor#6] [Ideas]
The reason I have commented out all the frame detection stuff is because I've switched the draw surface to `cairo.Surface` and would need to rewrite it to work with it. I therefore thought it would be better to use https://github.com/njean42/kumiko instead.

**Comment by GeoRW** [comment]
Yes, that's good idea. My algorithm is not that good and if there's specialized library for that now, let's use it.
